rb_ws/src/buggy/buggy/serial/ros_to_bnyahaj.py

```diff
@@ -160,16 +160,15 @@ def reader_thread(self):
                     )
 
             elif isinstance(packet, BnyaTelemetry):
-                self.get_logger().debug("packet" + str(packet))
                 odom = ROSOdom()
 
                 # TODO: Not mock rolled accurately (Needs to be Fact Checked)
                 try:
-                    lat, long = World.utm_to_gps(packet.x, packet.y)
+                    lat, long = utm.to_latlon(packet.x, packet.y, 17, "T")
```

Comment:
see, this is packet.x, packet.y, but top wasn't. Let's just mark this down and see what happens later.